amnesia.py

```diff
@@ -9,6 +9,12 @@
 from tweepy.error import TweepError
 
 
+# By default, do not actually delete any tweet/like.
+# Just print out what will be deleted.
+# Set this to False to actually delete tweets/likes.
+DRY_RUN = True
```

Comment:
Clarification. Add `dry_run = False` to `configs-sample.py` (such that it would get copied to `configs.py`). Then something like this in `amnesia.py`. Perhaps make the case for `twitter_config` and `dry_run` be consistent.

Does that make sense?

```
import config

twitter_config = config.twitter
dry_run = False
if hasattr(config, 'dry_run'):
    dry_run = config.dry_run
```